### PR TITLE
Show rounded value of volumes in sample list

### PR DESCRIPTION
**Pull request by GitEdvard**
Purpose:
Add function "showRounded" to show rounded values in lists. This function is located on a central place so that it may be referenced by all components.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/substances/substances.jsx [hunk 1]
```diff
@@ -92,7 +93,8 @@ class Substances extends React.Component {
       {
         Header: 'Volume',
         id: 'volume',
-        accessor: d => (d.properties.volume ? d.properties.volume.value : null),
+        accessor: d =>
```
Comment (withrocks) [inline src/sentry/static/sentry/app/views/substances/substances.jsx:96]: There is support for (localized) number formatting in the `Intl.NumberFormat` object which you might want to check out. It's supported in all relevant browsers:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
https://caniuse.com/#search=DateTimeFormat

**Comment by GitEdvard** [comment]
Now I've updated to use the Intl.NumberFormat method. The parameter locale=undefined should show number format according to settings in the local machine. However, I had problem testing this (I have to restart my machine and it is cumbersome :-(  ). Maybe we can merge this and possibly change it later if needed?

**Review by withrocks — APPROVED** [review]

**Comment by withrocks** [comment]
Sounds good. The CI build is failing now in the linter step: https://travis-ci.com/commonlims/commonlims/jobs/273920458

Are you not getting errors when you commit? If not I can come and take a look at why (should let you know if you don't commit with `--no-verify`)

**Comment by withrocks** [comment]
Turned out that the error is in the develop branch. I'll rebase and push a fix to this branch.

**Pull request merged**